sdk/rms_sdk/UnitTests/rest_clients_ut/SConscript

```diff
@@ -29,17 +31,15 @@ Import("""
 """)
 
 src_files = Glob('*.cpp')
-src_files += Glob(configuration + '/*.cpp')
+srcdir = Dir('.').srcnode().abspath
 
-rms_env.Append(CPPPATH='#sdk/rmscrypto_sdk/CryptoAPI')
+SetupUtMocs(srcdir, build_flavor)
 
-srcdir = Dir('.').srcnode().abspath.replace('\\','/')
-rms_env.Append(CPPDEFINES = { 'SRCDIR' : srcdir })
+src_files += Glob(build_flavor + '/*.cpp')
 
-lib_path = [
-    qt_lib_path,
-    '#bin/' + configuration + '/' + target_arch + '/sdk',
-]
+rms_env.Append(CPPPATH='#sdk/rmscrypto_sdk/CryptoAPI')
+
+rms_env.Append(CPPDEFINES = { 'SRCDIR' : "\\\"" + srcdir + "/\\\"" })
 
```

Comment:
these are applied to the whole rms_env. May be you need to replace this with ut-only env. Or move CPPPATH and CPPDEFINES to an outer Sconscript

Reply:
Created a new environment for these unit tests